source/MaterialXContrib/MaterialXNode/MayaUtil.cpp

```diff
@@ -0,0 +1,41 @@
+#include "MayaUtil.h"
+#include "Plugin.h"
+
+#include <maya/MViewport2Renderer.h>
+#include <maya/MFragmentManager.h>
+
+namespace MaterialXMaya
+{
+    void registerFragment(const std::string& fragmentName, const std::string& fragmentSource)
```

Comment:
The non-debug case. Removed all dependencies on MaterialX and MaterialXData in particular here.